src/main/java/com/mael/ttt/Game.java

```diff
@@ -2,27 +2,25 @@
 
 import com.mael.ttt.players.Player;
 
-import static com.mael.ttt.Mark.*;
-
 public class Game {
 
     private Turn turn;
-    private Player player;
-    private Player opponent;
+    private Player lastPlayer;
     private Player currentPlayer;
 
     public Game(Turn turn, Player player, Player opponent) {
-        this.turn     = turn;
-        this.player   = player;
-        this.opponent = opponent;
-        currentPlayer = player;
+        this.turn          = turn;
+        this.currentPlayer = player;
+        this.lastPlayer    = opponent;
     }
 
     public void start() {
-        while (turn.isNotGameOver()) {
+        while (turn.canBePlayed()) {
             turn.placeMark(getCurrentPlayer());
             swapPlayers();
         }
+
+        turn.printResults(lastPlayer.getMark());
```

Comment:
Much better now!

- you ask the question
- action
- and print the result

:boom: